scripts/prestart.sh

```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+# run from root of project
+
+# Let the DB start
+# python /app/app/backend_pre_start.py
+
+# Run migrations
+# alembic upgrade head
+
+# Create initial data in DB
+python3 -m app.initial_data
```

Comment:
ModuleNotFound error for python3 (sqlalchemy.orm). The env setup procedure sets up modules for 'python', but 'python3' is still pointing to some other version installed outside of the c3po environment. Recommend changing this line to python -m app.initial_data

Reply:
also having trouble confirming the initialization works via pgadmin (left a related note on the README)

Reply:
fixed python vs python3...will discuss pgadmin today